src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -33,9 +33,15 @@ public class KMeansPlusPlusTrainer : TrainerBase<RoleMappedData, KMeansPredictor
         public const string LoadNameValue = "KMeansPlusPlus";
         internal const string UserNameValue = "KMeans++ Clustering";
         internal const string ShortName = "KM";
-        internal const string Summary = "K-means is a popular clustering algorithm. With K-means, the data is clustered into a specified "
-            + "number of clusters in order to minimize the within-cluster sum of squares. K-means++ improves upon K-means by using a better "
-            + "method for choosing the initial cluster centers.";
+        internal const string Summary = "KMeans++ clustering algorithm";
```

Comment:
>internal const string Summary = "KMeans++ clustering algorithm"; [](start = 8, length = 64)

This summary isn't quite as useful. I understand that we're adding more descriptive text on remarks for the API, but this is a temporary solution as noted elsewhere, so why are we degrading the text that we do have?

Reply:
The other summaries  are just one liner, like: ''Averaged Perceptron Binary Classifier." or "Train a field-aware factorization machine for binary classification". 
This felt like the outlier.  Restoring it. 

---
In reply to: [199548768](https://github.com/dotnet/machinelearning/pull/455#discussion_r199548768) [](ancestors = 199548768)

Reply:
Well, I don't know. I think people don't write good summaries because they're lazy, I wouldn't want to normalize that laziness when someone actually did a good job writing a summary. :)